<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameAddOrderDismission</class>
 <widget class="QFrame" name="FrameAddOrderDismission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayoutOrderDismissalNumber">
       <item>
        <widget class="QLabel" name="labelOrderDismissalNumber">
         <property name="text">
          <string>Order Dismissal Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditOrderDismissalNumber"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayOutdateOfOrderDismissal">
       <item>
        <widget class="QLabel" name="labelOutdateOfOrderDismissal">
         <property name="text">
          <string>Outdate Of Order Dismissal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditOutdateOfOrderDismissal">
         <property name="displayFormat">
          <string notr="true">dd.MM.yyyy (ddd)</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAROD">
       <property name="text">
        <string>Add Row Order Dismissal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditOrderDismissalNumber</tabstop>
  <tabstop>pushButtonAROD</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
